### ISSUE TITLE
PXD-444 ⁃ support deleting property values for submitted node

### ISSUE THREAD

**Issue by philloooo** [uc-cdis/sheepdog#9]

**Comment by fortuno** [comment]
My suggestion would be:
- ignore (keep previous value): if that property is not in the TSV (no header).
- delete: if that property is in the submitted file with either empty or "null" value.

**Comment by dmiller15** [comment]
Agreed with Francisco on both points. For more on the second point, I agree as long as the deletion is done through the same submission system. The ideal situation would be to create a specific deletion option of some sort. Much like what we do with the node deletion there would be some sort of interface separate from submission that would allow people to remove individual properties.

**Comment by allisonheath** [comment]
Also might consider using `PATCH` and `PUT` verbs. Where `PUT` fully replaces everything that was there before and `PATCH` just updates whatever is specified. See: https://docs.cancergenomicscloud.org/docs/overwrite-a-files-metadata
